src/test/java/org/littil/api/exception/ThrowableMapperTest.java

```diff
@@ -16,13 +18,16 @@
 
 @QuarkusTest
 @TestHTTPEndpoint(GuestTeacherResource.class)
```

Comment:
@pschildkamp : should we use **@QuarkusTestResource(APIManagementMock.class)** here ?